.travis.yml

```diff
@@ -15,7 +15,7 @@ install:
 - travis_retry cypress install
 script:
 - npm run build
-- npm run test:coverage -- --runInBand
+- npm run test:coverage -- --runInBand --forceExit
```

Comment:
Awesome, thanks for this! Looks like it doesn't speed up our builds, but that's one less error